api/database/migrations/2024_12_18_133424_create_employee_profiles_table.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Query\Expression;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->jsonb('career_planning_organization_type_interest')->nullable();
+            $table->jsonb('career_planning_move_interest')->nullable();
+            $table->jsonb('career_planning_mentorship_status')->nullable();
+            $table->jsonb('career_planning_mentorship_interest')->nullable();
+            $table->boolean('career_planning_exec_interest')->nullable();
+            $table->jsonb('career_planning_exec_coaching_status')->nullable();
+            $table->jsonb('career_planning_exec_coaching_interest')->nullable();
+            $table->text('career_planning_about_you')->nullable();
+            $table->text('career_planning_career_goals')->nullable();
+            $table->text('career_planning_learning_goals')->nullable();
+            $table->text('career_planning_work_style')->nullable();
+
+            $table->string('dream_role_title')->nullable();
+            $table->text('dream_role_additional_information')->nullable();
+
+            $table->foreignUuid('dream_role_community_id')
+                ->nullable()
+                ->constrained(table: 'communities', column: 'id')
+                ->onDelete('cascade');
+            $table->foreignUuid('dream_role_classification_id')
+                ->nullable()
+                ->constrained(table: 'classifications', column: 'id')
+                ->onDelete('cascade');
+            $table->foreignUuid('dream_role_work_stream_id')
+                ->nullable()
+                ->constrained(table: 'work_streams', column: 'id')
+                ->onDelete('cascade');
+
+        });
+
+        Schema::create('department_user_dream_role', function (Blueprint $table) {
```

Comment:
The [issue](https://github.com/GCTC-NTGC/gc-digital-talent/issues/12001#:~:text=user_dream_role_departments%20%2D%20Table) has a differently named table, was this name change on purpose?

Reply:
I think I just wrote it wrong in the issue. This one follows the laravel convention of alphabetical table name ordering in pivot tables